src/adapters/fan.js

```diff
@@ -0,0 +1,127 @@
+const {ajax} = require('../ajax');
+const bidFactory = require('../bidfactory');
+const bidManager = require('../bidmanager');
+const BASE_URL = 'https://an.facebook.com/v1/placementbid.json';
+const ACCEPTED_FORMATS = ['320x50', '300x250', '728x90'];
+
+function sizesToFormats(sizes) {
+  return sizes
+    .map(([w, h]) => {
+      let format = `${w}x${h}`;
+      return ~ACCEPTED_FORMATS.indexOf(format) ? format : 'native';
```

Comment:
Is it possible to use `Array.prototype.includes` here? The IE polyfill fix in #464 should merge ahead of this PR.